storage/pv-to-path-mapping

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# map PVS to storage paths
+printf "%-40s\t%-40s\t%s\n" "PATH" "PV" "CLAIM"
+PVS=$(oc get pv -o jsonpath='{.items[*].metadata.name}' | tr ' ' "\n")
+declare -A PATH_TO_PV_MAP
+while read pv <&3; do
+  path=$(oc describe pv ${pv} | grep Path: | awk '{print $2}')
```

Comment:
I don't have OCS on a cluster to test but looking at the docs, jsonpath would be:
`jsonpath="{.spec.glusterfs.path}"`

Would that work instead of grepping?

Reply:
fixed.